create-snowpack-app/app-template-11ty/snowpack.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  install: [
+    /* ... */
+  ],
+  plugins: [
+    ['@snowpack/plugin-run-script', { cmd: 'eleventy', watch: '$1 --watch' }],
+  ],
+  installOptions: {
+    /* ... */
+  },
+  devOptions: {
+    hmrDelay: 300,
```

Comment:
Can you add a comment here saying "eleventy updates multiple files at once, so add a 300ms delay before we trigger a browser update".